specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,63 @@
+# Charge Subcategory
+
+Charge Subcategory indicates what kind of usage or adjustment the row represents. Charge Subcategory is a supplemental detail to the Charge Type. It provide additional context to describe the primary category of charge.
+
+This linkage to the parent ChargeType means that for every entry under ChargeType, there is a corresponding Charge Subcategory that further refines the nature of the charge. It's a nested level of detail that allows users to see not just what type of charge was incurred, but also how that charge interacts with their purchasing agreements and discounts.
```

Comment:
Seems like lines 5-9 could be summarized? feels like currently it's repeating a lot of stuff that's already said above. Below is a suggestion only that could be merged with the first line. It feels like listing sub category values for each parent value pattern won't be scalable when we look at upcoming subcategorizations for ServiceCategory etc.

```suggestion
This linkage to the parent Charge Type means that for every entry under Charge Type, there can be a corresponding Charge Subcategory that further refines the nature of the charge. It's a nested level of detail that allows users to see not just what type of charge was incurred. Current sub-categorization currently applies to Charge Type values 'Usage' and 'Adjustment'. Support for others will be added as needed.
```